spec/fixtures/hieradata/defined_type_specific_override.yaml

```diff
@@ -0,0 +1,5 @@
+---
+stunnel::connection::ssl_version: 'TLSv1'
```

Comment:
thought v1 was phased out and minimum was supposed to be 1.2 nowdays?

Reply:
@sharkbruhaha It's a test that works on both EL6 and EL7